integration/dvotingdela.go

```diff
@@ -137,7 +137,11 @@ func setupDVotingNodes(t require.TestingT, numberOfNodes int, tempDir string) []
 		delaNodes = append(delaNodes, node)
 		dVotingNodes = append(dVotingNodes, node)
 	}
-	delaNodes[0].Setup(delaNodes[1:]...)
+	if node != nil {
+		node.Setup(delaNodes...)
```

Comment:
the `node` name is a bit confusing, given there's a variable right above also called `node` which has a different meaning and values.